tests/aws/services/events/test_events_cross_account_region.py

```diff
@@ -0,0 +1,268 @@
+import json
+import time
+
+from localstack.testing.aws.util import is_aws_cloud
+from localstack.testing.pytest import markers
+from localstack.utils.strings import short_uid
+from tests.aws.services.events.helper_functions import sqs_collect_messages
+from tests.aws.services.events.test_events import EVENT_DETAIL, TEST_EVENT_PATTERN_NO_SOURCE
+
+SOURCE_PRIMARY = "source-primary"
+SOURCE_SECONDARY = "source-secondary"
+
+
+class TestEventCrossRegion:
+    @markers.aws.validated
+    def test_event_bus_to_event_bus_cross_region(self, aws_client_factory, cleanups, snapshot):
+        primary_region = "us-east-1"
+        secondary_region = "eu-central-1"
+
+        # overwriting randomized region https://docs.localstack.cloud/contributing/multi-account-region-testing/
+        # requires manually adding region replacement for snapshot
+        snapshot.add_transformer(snapshot.transform.regex(primary_region, "<region>"))
+        snapshot.add_transformer(snapshot.transform.regex(secondary_region, "<region>"))
+
+        aws_client_primary = aws_client_factory(region_name=primary_region)
+        aws_client_secondary = aws_client_factory(region_name=secondary_region)
```

Comment:
Here we can use: `aws_client` and `secondary_aws_client` fixtures.

Reply:
As discussed offline, for cross-region-only testing the factory method is currently the only option, since the `secondary_aws_client` fixture instantiates the client also for a second account.